features/step_definitions/requirements_steps.rb

```diff
@@ -0,0 +1,106 @@
+Before do
+  @substitutions = {}
+  @fields = {}
+end
+
+Given "I have created $type requirement" do |type|
+  page.visit("#{dm_frontend_domain}")
+
+  click_on "Find #{type}"
+
+  assert_selector('h1', text: "Find #{type}")
```

Comment:
have_selector is pretty much a wrapper for assert_selector, ill update calls to it.